src/timer_wrap.cc

```diff
@@ -62,7 +62,6 @@ class TimerWrap : public HandleWrap {
     env->SetProtoMethod(constructor, "hasRef", HandleWrap::HasRef);
 
     env->SetProtoMethod(constructor, "start", Start);
-    env->SetProtoMethod(constructor, "stop", Stop);
```

Comment:
I'm sure I've seen timer handles be used by modules in the wild, I wonder if the impact of this might be more problematic than it would seem at first glance?

Technically not "stable api", but this has existed pretty much forever.

Reply:
I did a scan a didn't find a single use of it. We've also made TimerWrap completely impossible to use by anything but our own timers because of the way OnTimeout works.

Reply:
Oh, right. 😕

Reply:
FWIW given that nobody complained when that happened, I think TimerWrap likely has 0 usage out there... :)